build_all/linux/build.sh

```diff
@@ -7,7 +7,6 @@ set -e
 script_dir=$(cd "$(dirname "$0")" && pwd)
 build_root=$(cd "${script_dir}/../.." && pwd)
 run_unit_tests=ON
-use_wsio=OFF
 run_valgrind=0
 build_folder=$build_root"/cmake/uamqp_linux"
 run_unittests=OFF
```

Comment:
Removed.

---
In reply to: [107511123](https://github.com/Azure/azure-uamqp-c/pull/58#discussion_r107511123) [](ancestors = 107511123)